remoteworkitem/trackeritem_repository.go

```diff
@@ -152,8 +143,7 @@ func upsert(db *gorm.DB, workItem app.WorkItem) (*app.WorkItem, error) {
 		}
 	} else {
 		log.Info(nil, map[string]interface{}{
-			"pkg":  "remoteworkitem",
-			"wiID": workItem.ID,
+			"pkg": "remoteworkitem",
```

Comment:
Can't we establish a habit of some sort that defines a `string` constant with the package name in the `doc.go` of a package for example? Then we can reuse that everywhere instead of having the package name repeated over and over. This actually makes changing the package name much hard than it should be.